Tests/iaas/flavor-naming/flavor-name-check.py

```diff
@@ -665,26 +704,31 @@ def main(argv):
     if argv[0] == "-1":
         prefer_old = True
         argv = argv[1:]
+    if argv[0] == "-3":
+        v3_flv = True
+        argv = argv[1:]
     if argv[0] == "-o":
         accept_old_mand = True
         argv = argv[1:]
     if argv[0] == "-c":
         completecheck = True
-        scsMandatory = readmandflavors(mandFlavorFile)
+        scsMandatory, scsRecommended = readflavors(mandFlavorFile, v3_flv, prefer_old)
         scsMandNum = len(scsMandatory)
+        scsRecNum = len(scsRecommended)
         if debug:
             print(f"Check for completeness ({scsMandNum}): {scsMandatory}")
         argv = argv[1:]
     if argv[0] == "-C":
         completecheck = True
-        scsMandatory = readmandflavors(argv[1])
+        scsMandatory, scsRecommended = readflavors(argv[1], v3_flv, prefer_old)
         scsMandNum = len(scsMandatory)
+        scsRecNum = len(scsRecommended)
         if debug:
             print(f"Check for completeness ({scsMandNum}): {scsMandatory}")
         argv = argv[2:]
 
     # Interactive input of flavor
-    if argv[0] == "-i":
+    if len(argv) and argv[0] == "-i":
```

Comment:
Just a bit more "pythonic":
```suggestion
    if argv and argv[0] == "-i":
```